python/sanlock.c

```diff
@@ -46,6 +46,20 @@ of the GNU General Public License v2 or (at your option) any later version.");
 /* Sanlock exception */
 static PyObject *py_exception;
 
+/*
+    Python 2 forward compatibility functions with Python 3
+    TODO: Once Python 2 will reach EOL at 2020 discrad this section
+*/
+#if PY_MAJOR_VERSION == 2
+
+static const char*
+PyUnicode_AsUTF8(PyObject* obj)
+{
+    return PyString_AsString(obj);
+}
```

Comment:
This does not convert to utf8 on python 2.

What is the purpose of this call? Lets find a good function name describing what it does
and add a helper hiding the ifdef - insead of wrapping a helper with the ifdef.